src/Resources/contao/languages/en/explain.xlf

```diff
@@ -78,7 +78,7 @@
         <source>Pixel densities/&lt;br&gt;scale factors</source>
       </trans-unit>
       <trans-unit id="XPL.imageSizeDensities.1.1">
-        <source>If the sizes attribute is not defined, this setting simply defines the pixel densities you want to support. The dimensions of the images are adjusted automatically. E.g. &lt;code&gt;1x, 1.5x, 2x&lt;/code&gt; creates the following HTML code:&lt;br&gt;&lt;code&gt;&amp;lt;img srcset="img-a.jpg 1x, img-b.jpg 1.5x, img-c.jpg 2x"&amp;gt;&lt;/code&gt;&lt;br&gt;&lt;br&gt;If the sizes attribute is defined, the same image sizes get generated but width descriptors are used for the srcset attribute. E.g. a 200 pixel wide image with the densities &lt;code&gt;1x, 1.5x, 2x&lt;/code&gt; creates the following HTML code:&lt;br&gt;&lt;code&gt;&amp;lt;img srcset="img-a.jpg 200w, img-b.jpg 300w, img-c.jpg 400w"&amp;gt;&lt;/code&gt;&lt;br&gt;&lt;br&gt;For more information about the srcset attribute please visit &lt;a href="http://www.w3.org/html/wg/drafts/html/master/embedded-content.html#attr-img-srcset" target="_blank"&gt;w3.org&lt;/a&gt;.</source>
+        <source>If the sizes attribute is not defined, this setting simply defines the pixel densities you want to support. The dimensions of the images are adjusted automatically. E.g. &lt;code&gt;1x, 1.5x, 2x&lt;/code&gt; creates the following HTML code:&lt;br&gt;&lt;code&gt;&amp;lt;img srcset="img-a.jpg 1x, img-b.jpg 1.5x, img-c.jpg 2x"&amp;gt;&lt;/code&gt;&lt;br&gt;&lt;br&gt;If the sizes attribute is defined, the same image sizes get generated but width descriptors are used for the srcset attribute. E.g. a 200 pixel wide image with the densities &lt;code&gt;200w, 300w, 400w&lt;/code&gt; creates the following HTML code:&lt;br&gt;&lt;code&gt;&amp;lt;img srcset="img-a.jpg 200w, img-b.jpg 300w, img-c.jpg 400w"&amp;gt;&lt;/code&gt;&lt;br&gt;&lt;br&gt;For more information about the srcset attribute please visit &lt;a href="http://www.w3.org/html/wg/drafts/html/master/embedded-content.html#attr-img-srcset" target="_blank"&gt;w3.org&lt;/a&gt;.</source>
```

Comment:
I think “width descriptors” is the term used by the HTML specification. Should I change it to “with the width descriptors `200w, 300w, 400w`”?